kitsune/messages/utils.py

```diff
@@ -18,38 +17,58 @@ def send_message(to, text=None, sender=None):
     # We need a sender, a message, and someone to send it to
     if not sender or not text or not to:
         return
-    # This is users from the To field
-    users = {user for user in to if isinstance(user, User)}
-    # This is groups from the To field
-    groups = {group for group in to if isinstance(group, Group)}
-    # This is all the users that are going to receive the message
-    receivers = users | set(User.objects.filter(groups__in=groups).distinct())
+    # User pks from users in To field
+    usernames = []
+    group_names = []
 
-    outbox_message = OutboxMessage.objects.create(sender=sender, message=text)
-    # Add the users from the To field to the outbox message, not all the users
-    # that are going to receive the message - this way we don't overwhelm the
-    # message UI
-    outbox_message.to.set(users)
+    for obj in to:
+        if isinstance(obj, User):
+            usernames.append(obj)
```

Comment:
Since we want the actual `User` objects later for setting the `to` field of the `OutboxMessage`, can't we just collect those here -- in addition to the `usernames` above -- by creating another list called `users` and doing a `users.append(obj)` here?

Reply:
I think this is bad naming on behalf of a cut and paste - these are objects being collected, not names. Even my comment is broken here - I'll fix this up. But we get the objects separated here.